fission/src/ui/modals/configuring/ChangeInputsModal.tsx

```diff
@@ -343,173 +345,127 @@ const ChangeInputsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
-
-                                                            {/* // Button to switch between two buttons and a joystick axis */}
-                                                            <Checkbox
-                                                                label="Use Buttons"
-                                                                defaultState={c.useGamepadButtons}
-                                                                onClick={val => {
-                                                                    setUseButtons(prevState => ({
-                                                                        ...prevState,
-                                                                        [c.inputName]: val,
-                                                                    }))
-                                                                    c.useGamepadButtons = val
-                                                                }}
-                                                            />
-                                                            {/* // Button to invert the joystick axis */}
-                                                            <Checkbox
-                                                                label="Joystick Inverted"
-                                                                defaultState={c.joystickInverted}
-                                                                onClick={val => {
-                                                                    c.joystickInverted = val
-                                                                }}
-                                                            />
-                                                        </div>
-                                                    )
-                                                }
-                                            }
-                                        })}
-                                    </Stack>
+                                    <Checkbox
+                                        label="Use Controller"
+                                        defaultState={selectedScheme?.usesGamepad ?? false}
+                                        onClick={val => {
+                                            setUseGamepad(val)
+                                            if (selectedScheme) selectedScheme.usesGamepad = val
+                                        }}
+                                    />
+                                    <Label size={LabelSize.Medium}>Default Control Schemes</Label>
+                                    <Button
+                                        value={"Reset all to Defaults"}
+                                        onClick={() => {
+                                            openModal("reset-inputs")
+                                        }}
+                                    />
```

Comment:
Could you move this to the bottom of the choose input scheme panel because it seems as though it only resets the control scheme